packages/lodestar/src/network/peers/discover.ts

```diff
@@ -127,6 +127,8 @@ export class PeerDiscovery {
     await this.discv5.start();
     this.discv5StartMs = Date.now();
     this.discv5.on("discovered", this.onDiscovered);
+    // on start, dial the discv5.bootEnrs that has been added to the routing table
```

Comment:
I think we have two choices here:
1. only do this if network.connectToDiscv5Bootnodes is true
2. remove the network.connectToDiscv5Bootnodes option entirely

Reply:
We are not supposed to connect to the Mainnet bootnodes, this has to be behind a cli flag and used only for local devnets

Reply:
I have added it behind the `network.connectToDiscv5Bootnodes` cli flag